client/src/pages/Stories/Stories.css

```diff
@@ -0,0 +1,17 @@
+h1 {
+    margin-top: 5%;

```

Comment:
Haven't had a chance to merge it yet, but Maggie added a global padding-top to the page so our elements won't overlap with the header. Might be able to remove this line.